qutebrowser/utils/utils.py

```diff
@@ -810,6 +811,11 @@ def get_clipboard(selection=False):
         mode = QClipboard.Selection if selection else QClipboard.Clipboard
         data = QApplication.clipboard().text(mode=mode)
 
+    target = "Primary selection" if selection else "Clipboard"
+    if not data.strip():
+        raise cmdexc.CommandError("{} is empty.".format(target))
```

Comment:
Would a `CommandError` be appropriate to raise when parsing `{clipboard}` and `{primary}`?